app.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+
+data = [['a','b','c'], ['1','2','3','4']]
```

Comment:
**Suggestion:** Handle the possibility of an empty data list in createDataFrame().
```suggestion
data = [['a','b','c'], ['1','2','3','4']] if data else []
```